src/test/java/com/exactpro/th2/mstore/TestMessagePersistor.java

```diff
@@ -98,6 +107,8 @@ void setUp() throws InterruptedException, CradleStorageException {
                 .build();
         persistor = spy(new MessagePersistor(errorCollector, storageMock, config));
         persistor.start();
+
+        reset(persistor);
```

Comment:
Why do you need to call the `reset` function here? We create a `persister` for each test. Why do we have to reset after we just created it?

Reply:
Because I call persistor.start();

Reply:
Okay...